plugins/ReverbSC/CMakeLists.txt

```diff
@@ -7,18 +7,21 @@ SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
 
 BUILD_PLUGIN(
 	reverbsc
-	ReverbSC.cpp 
-	ReverbSCControls.cpp 
-	ReverbSCControlDialog.cpp 
+	ReverbSC.cpp
+	ReverbSCControls.cpp
+	ReverbSCControlDialog.cpp
 	base.c
 	revsc.c
+	base.h
+	revsc.h
+	dcblock.h
 	dcblock.c
-	MOCFILES 
 	ReverbSC.h
-	ReverbSCControls.h 
-	ReverbSCControlDialog.h 
+	MOCFILES
+	ReverbSCControls.h
+	ReverbSCControlDialog.h
 	base.h
 	revsc.h
 	dcblock.h
```

Comment:
This is because `git` sometimes do wrong merges, need to correct this by doing a new `git merge` action (cleaner approach)